### PR TITLE
Add #normalizeCamelCase and test

### PR DESCRIPTION
**Pull request by hernanmd**
Add method to split a camel cased String into its "normalized" string version, separated by spaces.

```smalltalk
'PinkFloyd' normalizeCamelCase >>> 'Pink Floyd'
```

### CODE REVIEW

File: src/Collections-Strings-Tests/StringTest.class.st [hunk 1]
```diff
@@ -1654,6 +1654,17 @@ StringTest >> testMatch [
...
 
+{ #category : #tests }
+StringTest >> testNormalizeCamelCase [
+
+	self assert: '' normalizeCamelCase equals: String empty.
+	self assert: 'CamelCase' normalizeCamelCase equals: 'Camel Case'.
+	self assert: 'CamelCase123' normalizeCamelCase equals:'Camel Case123'.
+	self assert: 'ABC' normalizeCamelCase equals: 'ABC'.
+	self assert: 'TresTristesTigres' normalizeCamelCase equals: 'Tres Tristes Tigres'.
+	self assert: 'TresTristesTigresABC' normalizeCamelCase equals: 'Tres Tristes Tigres ABC'
```
Comment (astares) [inline src/Collections-Strings-Tests/StringTest.class.st]: please add at least one test with two wording parts as input **separated by space** to cover that case

self assert: 'CamelCase HelloWorld' normalizeCamelCase equals: ...

File: src/Collections-Strings/String.class.st [hunk 2]
```diff
@@ -2178,6 +2178,19 @@ String >> match: text [
 	^ self startingAt: 1 match: text startingAt: 1
 ]
 
+{ #category : #converting }
+String >> normalizeCamelCase [
+	"
+	'TheRollingStones' normalizeCamelCase >>> 'The Rolling Stones'
+	"
+	^ self class streamContents: [ :out |
+		self do: [ :e |
```
Comment (astares) [inline src/Collections-Strings/String.class.st]: why do you call the block parameter "out" and not "s" (like Stream) or better "stream" 
why do you call the block parameter "e" and not "c" (like Character) or better "char" or even better "character"

**Review by astares — CHANGES_REQUESTED** [review]

File: src/Collections-Strings/String.class.st (hunk 2 above)
Comment (hernanmd) [reply src/Collections-Strings/String.class.st]: Ok, good suggestions, I just integrated them and fixed the case to not write two consecutive blank spaces where there is a single space in the input string.

File: src/Collections-Strings-Tests/StringTest.class.st (hunk 1 above)
Comment (hernanmd) [reply src/Collections-Strings-Tests/StringTest.class.st]: Added

**Review by astares — APPROVED** [review]

**Pull request merged**